homeassistant/components/media_player/enigma2.py

```diff
@@ -0,0 +1,220 @@
+
+        if self.e2_box.get_current_playback_type(
+                self.currservice_serviceref) == PlaybackType.recording:
+            return SUPPORT_ENIGMA2_RECORDING_PLAYBACK
+
+        return SUPPORT_ENIGMA2_LIVE_TV
+
+    def turn_off(self):
+        """Turn off media player."""
+        if self.state is STATE_ON:
+            self.e2_box.toggle_standby()
+
+    def turn_on(self):
+        """Turn the media player on."""
+        if self.state is STATE_OFF:
+            self.e2_box.toggle_standby()
+            self.update()
+
+    @property
+    def media_title(self):
+        """Title of current playing media."""
+        if self.currservice_serviceref.startswith('1:0:0'):
+            return "[Recording Playback] - " + \
+                   self.current_programme_name
+        if self.current_service_channel_name:
+            return self.current_programme_name + " - " + \
+                   self.current_service_channel_name
+
+        return self.current_programme_name
+
+    @property
+    def media_channel(self):
+        """Channel of current playing media."""
+        return self.current_service_channel_name
+
+    @property
+    def media_content_id(self):
+        """Service Ref of current playing media."""
+        return self.currservice_serviceref
+
+    @property
+    def media_content_type(self):
+        """Type of video currently playing."""
+        return MEDIA_TYPE_TVSHOW
+
+    @property
+    def is_volume_muted(self):
+        """Boolean if volume is currently muted."""
+        return self.muted
+
+    @property
+    def media_image_url(self):
+        """Picon url for the channel."""
+        return self.picon_url
+
+    def set_volume_level(self, volume):
+        """Set volume level, range 0..1."""
+        self.e2_box.set_volume(int(volume * 100))
+        self.volume = volume * 100
+
+    def volume_up(self):
+        """Volume up the media player."""
+        self.volume += 5
+        self.e2_box.set_volume(self.volume)
+
+    def volume_down(self):
+        """Volume down media player."""
+        self.volume -= 5
+        self.e2_box.set_volume(self.volume)
+
+    @property
+    def volume_level(self):
+        """Volume level of the media player (0..1)."""
+        _LOGGER.info('self.volume : %s', self.volume)
+        return self.volume / 100
+
+    def media_play_pause(self):
+        """Pause media on media player."""
+        self.e2_box.toggle_play_pause()
+
+    def media_play(self):
+        """Play media."""
+        self.e2_box.toggle_play_pause()
+
+    def media_pause(self):
+        """Pause the media player."""
+        self.e2_box.toggle_play_pause()
+
+    def media_next_track(self):
+        """Send next track command."""
+        self.e2_box.set_channel_up()
+
+    def media_previous_track(self):
+        """Send next track command."""
+        self.e2_box.set_channel_down()
+
+    def mute_volume(self, mute):
+        """Send mute command."""
+        self.e2_box.mute_volume()
```

Comment:
The `mute` parameter is a boolean saying if the muting should be true/on or false/off. Please implement that.